apps/openmw/mwworld/worldimp.cpp

```diff
@@ -3245,9 +3244,7 @@ namespace MWWorld
 
     float World::feetToGameUnits(float feet)
     {
-        // Looks like there is no GMST for this. This factor was determined in experiments
-        // with the Telekinesis effect.
-        return feet * 22;
+        return feet * Constants::UnitsPerFoot;
```

Comment:
Now it is 21.33333333. Are we fine with it (since it is an arbitrary value, according to comment), or we desperately need a 22 here?

Reply:
Not sure. Do a test? The function is used in only a few places. Shouldn't be hard to figure out if there is a noticeable difference.